Documentation/ReleasePlan.md

```diff
@@ -0,0 +1,43 @@
+# Release Plan
+
+## Versioning strategy
+
+Coverlet is versioned with Semantic Versioning [2.0.0](https://semver.org/#semantic-versioning-200) that states:
+
+```
+Given a version number MAJOR.MINOR.PATCH, increment the:
+
+MAJOR version when you make incompatible API changes,
+MINOR version when you add functionality in a backwards-compatible manner, and
+PATCH version when you make backwards-compatible bug fixes.
+Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+```
+
+## Release Calendar
+
+We release 3 components as nuget packages:  
+
+**coverlet.msbuild.nupkg**  
+**coverlet.console.nupkg**  
+**coverlet.collector.nupkg**  
+
+We plan 1 release every [Q1](https://en.wikipedia.org/wiki/Calendar_year) if there is *at least* 1 new commit of source code on master.  
```

Comment:
Do you mean 1 _major_ release?

Reply:
No I mean if we "touch" code we'll release something...it's unrelated to MAJOR.MINOR choice.
For instance
* ver 1.0.0 we do 100 code commit, simple fix, after quarter we release 1.0.100
* ver 10.0.0 we do 0 code commit we don't release anything and we'll wait next quarter

Feel free to fix my poor english.

Reply:
```suggestion
We plan 1 release [once per quarter](https://en.wikipedia.org/wiki/Calendar_year) if there is *at least* 1 new commit of source code on master. This release may be a major, minor, or patch version upgrade from the previous release depending on impact to consumers. 
```